docs/whats-new/preview-5.md

```diff
@@ -0,0 +1,595 @@
+                "`Session`"
+            );
+        }
+    )
+    .AddDatabase("inventory");
+#pragma warning restore ASPIRE001
+
+builder.AddProject<Projects.InventoryApi>("inventoryapi")
+       .WithReference(cosmos);
+```
+
+This can be applied globally in your project or in a more localized way as shown above. For more information, see [.NET Aspire diagnostics overview](../diagnostics/overview.md).
+
+### Azure provisioning for local development
+
+Previous .NET Aspire previews have had limited support for using cloud based resources for local development. You could use an emulator (such as Azurite for Azure Storage) or you could provision a real resource in the cloud and place a connection string in your AppHost's user secrets.
+
+In preview 5 if you want to use an Azure resource where an emulator does not exist you can add
+the following settings to user `secrets.json` file:
+
+```json
+{
+  "Azure": {
+    "SubscriptionId": "<subscription id>",
+    "Location": "<default location>",
+    "ResourceGroup": "<resource group name>"
+  }
+}
+```
+
+When you launch the AppHost the dashboard will show that it is create the Azure resources for you and provide helpful links to the deployment into the Azure portal, or in the case of failure logs that provide hints as to what might be causing the deployment issue.
+
+:::image type="content" source="/media/preview-5/azure-resource-provisioning-on-dashboard.png" lightbox="/media/preview-5/azure-resource-provisioning-on-dashboard.png" alt-text=".NET Aspire dashboard: Azure provisioning.":::
+
+To use Azure provisioning you only need to make use of one of an Azure resource. For example:
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var blobs = builder.AzureStorage("storage")
+                   .AddBlobs("blobs");
+
+builder.AddProject<Projects.GalleryApp>("galleryapp")
+       .WithReference(blobs);
+```
+
+Some resources in .NET Aspire such as Postgres are not cloud specific and can run locally in a container but when deployed use a managed service (such as Azure Postgres Flexible Server).
+
+```csharp
+var builder = DistributedApplication.CreateBuilder(args);
+
+var db = builder.AddPostgres("pgsql")
+                .PublishAsAzurePostgresFlexibleServer()
+                .AddDatabase("inventorydb");
+
+builder.AddProject<Projects.InventoryApi>("inventoryapi")
+       .WithReference(db);
+```
+
+It is also possible to use an Azure hosted resource for local development by using the `AsAzurePostgresFlexibleServer()` extension method instead. When this method is used the container will not be started locally and cloud-based instance will be created just like the Azure only resource types. The `PublishAsX` and `AsX` methods also support callbacks to customize the underlying Azure resources as shown above in the Cosmos DB example.
+
+For more information, see [Local Azure provisioning](../deployment/azure/local-provisioning.md).
+
+## Other Azure-related improvements
```

Comment:
This heading feels duplicative -- right above the heading is "Azure Improvements" then we list some, then we have a heading "Other Azure improvements" -- I know these are a mixture of a few things, but they still are Azure Improvements. Recommend scrapping this heading/intro. Thoughts @IEvangelist